core/app/api/customer/groups/route.ts

```diff
@@ -0,0 +1,41 @@
+import { NextResponse } from 'next/server';
+
+import { CustomerGroupsSchema } from '~/lib/makeswift/components/customer-group-slot/schema';
+
+export async function GET() {
+  if (!process.env.BIGCOMMERCE_ACCESS_TOKEN) {
+    // eslint-disable-next-line no-console
+    console.error('BIGCOMMERCE_ACCESS_TOKEN is not set');
+
+    return NextResponse.json({ status: 'error', error: 'Internal server error' }, { status: 500 });
```

Comment:
This can be replaced with a thrown error. It will be logged to the console and remove the need to disable the ESLint rule and manually craft an error response